lib/core_ext/numpy.rb

```diff
@@ -59,6 +73,44 @@ def roll(shift, axis: 0)
     end
   end
 
+  def assign(x1, y1, x2, y2, value)
```

Comment:
not crazy about the name. maybe use `full` to match the existing method? I think you could also accomplish the same thing with `paste(... Array.full(...) ...)` so maybe we don't need this at all.